core/src/cluster_info.rs

```diff
@@ -95,6 +93,8 @@ const MAX_GOSSIP_TRAFFIC: usize = 128_000_000 / PACKET_DATA_SIZE;
 
 /// Keep the number of snapshot hashes a node publishes under MAX_PROTOCOL_PAYLOAD_SIZE
 pub const MAX_SNAPSHOT_HASHES: usize = 16;
+/// Lookback window duration of bandwidth usage of pull requests.
+const BANDWIDTH_USAGE_LOOKBACK_DURATION: Duration = Duration::from_secs(28800);
```

Comment:
Any particular reason to use this specific time ~8 hours?

Reply:
not really, just a placeholder. what do you think would make sense here?